local-test-configuration/monitoring/index.html

```diff
@@ -858,6 +865,23 @@ <h2>Active data feeds</h2>
       return sponsorWallet;
     };
 
+    const deriveSponsorWallet = (
+      sponsorWalletMnemonic,
+      dapiNameOrDataFeedId,
+      updateParameters,
+      walletDerivationScheme
+    ) => {
+      // Derive the hash of the data feed using which the sponsor wallet is derived. For self-funded feeds it's more
```

Comment:
This is a bit strange, but it's super minor (especially seeing as it's part of the _monitoring_ for the test configuration.

Reply:
Yeah, the comment is a bit confusing. I've tried rewording it to better reflect what's implemented. Let me know if it makes sense.